oximeter/oximeter/schema/instance-data-link.toml

```diff
@@ -0,0 +1,86 @@
+format_version = 1
+
+[target]
+name = "instance_data_link"
+description = "A network data link related to a guest/instance"
```

Comment:
This seems a bit confusing. For example, what does "related" mean? My 2c is: "A network data link attached to a virtual machine instance" or similar.